Tests/TuistLoaderTests/SwiftPackageManager/PackageInfoMapperTests.swift

```diff
@@ -33,7 +33,8 @@ final class PackageInfoMapperTests: TuistUnitTestCase {
         let basePath = try temporaryPath()
         try fileHandler.createFolder(basePath.appending(try RelativePath(validating: "Sources/Target_1")))
         try fileHandler.createFolder(basePath.appending(try RelativePath(validating: "Sources/Target_2")))
-        let resolvedDependencies = try subject.resolveExternalDependencies(
+        let resolvedDependencies = try await subject.resolveExternalDependencies(
+            path: .current,
```

Comment:
I'd recommend using a `temporaryPath()` instead of `.current` since that variable uses `FileManager.default` directly and is different based on your environment, making these tests less stable